mainapp/Classes/Managers/LanguageManager.cpp

```diff
@@ -304,94 +304,94 @@ void LanguageManager::initLocalizationMap()
     _localizationMapEnglish["Math"] = "»ç‡æÌ"; // गणित
     _localizationMapSwahili["Math"] = "Hesabu";
 
-    _localizationMapEnglish["TutorialTrace"] = "ÚUð¹æ ¥ÙéÚUð¹‡æ"; // रेखा अनुरेखण
+    _localizationMapEnglish["TutorialTrace"] = "ÚUð¹æ ¥ÙéÚUð¹‡æ\nLine Tracing"; // रेखा अनुरेखण
```

Comment:
Any specific reason for reversing the English and Hindi text?
In the above file (strings.xml), English text is followed by Hindi text

Reply:
It has no specific reason. It is anyway handled in setPosition function